workflow-dev/tpl/deis-workflow-manager-rc.yaml

```diff
@@ -25,19 +25,19 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: WORKFLOW_MANAGER_PORT
+        - name: PORT
           value: "8080"
-        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+        - name: VERSIONS_API_URL
           value: {{.workflowManager.versionsApiURL}}
-        - name: WORKFLOW_MANAGER_DOCTORAPIURL
+        - name: DOCTOR_API_URL
```

Comment:
wondering if we want to follow a similar pattern as `VERSIONS_API_URL` above?  If so, we can allow release charts go out with the prod `DOCTOR_API_URL` (and figure we'd have this value be the staging url for the `dev`/non-release charts...